universal-application-tool-0.0.1/app/views/dev/DatabaseSeedView.java

```diff
@@ -36,11 +37,23 @@ public DatabaseSeedView(BaseHtmlLayout layout, ObjectMapper objectMapper) {
 
   public Content render(
       Request request,
-      ImmutableList<ProgramDefinition> programDefinitions,
+      ActiveAndDraftPrograms activeAndDraftPrograms,
       ImmutableList<QuestionDefinition> questionDefinitions,
       Optional<String> maybeFlash) {
 
-    String prettyPrograms = getPrettyJson(programDefinitions);
+    ImmutableList<ProgramDefinition> draftPrograms =
+        activeAndDraftPrograms.getProgramNames().stream()
+            .filter(name -> activeAndDraftPrograms.getDraftProgramDefinition(name).isPresent())
+            .map(name -> activeAndDraftPrograms.getDraftProgramDefinition(name).get())
+            .collect(ImmutableList.toImmutableList());
+    ImmutableList<ProgramDefinition> activePrograms =
```

Comment:
I guess I'd probably add this as a method on ActiveAndDraftPrograms, but it's all the same to me.

Reply:
uh, sorry, in our normal language that would be:
nit(nonblocking): this might be good as a method on ActiveAndDraftPrograms (maybe ActiveAndDraftQuestions also).

Reply:
Cool, can do.